F1WM.UnitTests/Services/TracksServiceTests.cs

```diff
@@ -55,5 +55,32 @@ public async Task ShouldGetTrackRecordsUntilSpecifiedYear()
 			repositoryMock.Verify(r => r.GetTrackRecords(trackId, trackVersion, year), Times.Once);
 			actual.Should().BeEquivalentTo(records);
 		}
+
+		[Fact]
+		public async Task ShouldGetTracks()
+		{
+			var page = 2;
+			var countPerPage = 21;
+			var tracks = fixture.Create<PagedResult<TrackSummary>>();
+			repositoryMock.Setup(r => r.GetTracks(page, countPerPage)).ReturnsAsync(tracks);
```

Comment:
Please verify if data is correct, i.e. the tracks that are returned by the repository are returned by the service as well.